src/languages/bnf.js

```diff
@@ -0,0 +1,29 @@
+/*
+ Language: Backus–Naur Form
+ Author: Oleg Efimov <efimovov@gmail.com>
+ */
+
+function(hljs){
+  return {
+    contains: [
+      // Attribute
+      {
+        className: 'attr',
+        begin: /</,
+        end: />/
+      },
```

Comment:
Oh, and I think they are should be 'attribute', not just 'attr', as they define structure. In other words, 'attr' is for something small and simple, like HTML attributes inside tags, and 'attribute' is for something like properties in CSS.